.github/workflows/openshift-upgrade-test.yaml

```diff
@@ -32,7 +32,9 @@ concurrency:
 jobs:
   e2e-tests:
     name: Prepare E2E configuration and image
+    if: ${{  vars.OPENSHIFT_SKIP != 'true'  }}
     runs-on: ubuntu-latest
+    environment: test
```

Comment:
It did not skip it:
https://github.com/mongodb/mongodb-atlas-kubernetes/actions/runs/8021707947/job/21914395620

I honestly do not understand how this is not working